### ISSUE TITLE
Feature Request: Optionally abort test run if number of failures exceeds threshold

### ISSUE THREAD

**Issue by bergmeister** [microsoft/vstest#1901]
I have looked at the documentation and could not find an option to abort a test run early on if the number of test failures exceeds a threshold.
This would be useful for scenarios where a bad code change is causing a long test run because the majority of tests fail, therefore leading to a very long test run. In most CI systems, one can see results only after completion of a task. This would be helpful to fail quicker.
As a user it would be nice to have this optional feature.

**Comment by mayankbansal018** [comment]
@cltshivash , @PBoraMSFT we have something similar in Azure devops pipeline right?

**Comment by bergmeister** [comment]
As far as I am aware Azure pipelines have the ability to re-run test(s) if they failed but I could not find an option to abort the whole test suite if there are too many test failures

**Comment by nohwnd** [comment]
This is a new feature and won't be implemented, we are focusing on adding new features to Testing.Platform instead. https://aka.ms/testingplatform
